pkg/agent/baker.go

```diff
@@ -964,7 +964,7 @@ func getPortRangeEndValue(portRange string) int {
 // they typically use GRID, not CUDA drivers, and will fail to install CUDA drivers.
 // NVv1 seems to run with CUDA, NVv5 requires GRID.
 // NVv3 is untested on AKS, NVv4 is AMD so n/a, and NVv2 no longer seems to exist (?).
```

Comment:
let's add a comment here to explicitly point out that karpenter is using this new API